source/getting-started/autostart-upstart.markdown

```diff
@@ -9,23 +9,47 @@ sharing: true
 footer: true
 ---
 
-Many linux distributions use the Upstart system (or similar) for managing daemons. Typically, systems based on Debian 7 or previous use Upstart. This includes Ubuntu releases before 15.04. If you are unsure if your system is using Upstart, you may check with the following command:
+Many linux distributions use the <a href="http://upstart.ubuntu.com/cookbook/" target="_blank">Upstart</a> system (or similar) for managing daemons. Typically, systems based on Debian 7 or previous use Upstart. This includes Ubuntu releases before 15.04. If you are unsure if your system is using Upstart, you may check with the following command:
 
 ```bash
 $ ps -p 1 -o comm=
 ```
 
-If the preceding command returns the string `init`, you are likely using Upstart.
+If the preceding command returns the string `init`, you are likely using Upstart. Just to be really sure, request the Upstart version with:
 
-Upstart will launch init scripts that are located in the directory `/etc/init.d/`. A sample init script for systems using Upstart is <a href="https://raw.githubusercontent.com/home-assistant/home-assistant/dev/script/hass-daemon">maintained by this project</a>.
+```
+$ init --version
+```
+
+The output should mention something similar to `init (upstart 1.12.1)`. If it doesn't, you're dealing with a traditional (SysV-style) init system and you should follow [these instructions](../autostart-init).
+
+Upstart will use configuration files in `/etc/init/`. A sample Upstart configuration is included in the <a href="https://github.com/home-assistant/home-assistant/blob/dev/script/hass.conf">Home Assistant source code</a>.
+
+Installation
+-------------
+Create a file named `/etc/init/hass.conf` with the following contents:
+
+```
```

Comment:
It isn't shell either, it's an upstart configuration file. So I think general code block would apply.